org.civicrm.reqangular/src/common/models/contact.js

```diff
@@ -157,6 +157,16 @@ define([
           return contactAPI.find(id).then(function (contact) {
             return instance.init(contact, true);
           });
+        },
+
+        /**
+         * Finds all the contacts managed by the sent contact id
+         *
+         * @param {string} id - contact id
+         * @return {Promise} - Resolves with found contacts/API Errors
+         */
+        leaveManagees: function (id, params) {
```

Comment:
We also have this in the instance, as it also makes sense to say `contact.leaveManagees()` and it will only load his managees. But in certain scenarios we also might need to get the managees when we only know the contact id. So added it to model.